OnDemandPassengerTests/VenueServiceTests.swift

```diff
@@ -6,16 +6,26 @@
 import XCTest
 
 class VenueServiceTests: XCTestCase {
-  var vc: SearchViewController!
+  var sharedVenueService: VenueService?
   
   override func setUp() {
     super.setUp()
     
-    let storyboard = UIStoryboard(name: "Main", bundle: Bundle.main)
-    vc = storyboard.instantiateViewController(withIdentifier: "SearchViewController") as! SearchViewController
+    sharedVenueService = VenueService()
   }
   
   func testPlacesRequestTriggeredProperly() {
```

Comment:
The name does not match the test. This is checking that the venue service requests the signature, right?